lib/rules/a11y-svg-has-accessible-name.js

```diff
@@ -0,0 +1,41 @@
+const {hasProp} = require('jsx-ast-utils')
+const {getElementType} = require('../utils/get-element-type')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'SVGs must have an accessible name',
+      url: require('../url')(module),
+    },
+    schema: [],
+  },
+
+  create(context) {
+    return {
+      JSXOpeningElement: node => {
+        const elementType = getElementType(context, node)
+
+        // Check if there is a nested title element that is the first child of the `<svg>`
+        const hasNestedTitleAsFirstChild =
+          node.parent.children.length &&
+          node.parent.children[0].type === 'JSXElement' &&
+          node.parent.children[0].openingElement.name.name === 'title'
```

Comment:
May want to guard these which will prevent errors on property access:

```suggestion
        const hasNestedTitleAsFirstChild =
          node.parent.children?.[0]?.type === 'JSXElement' &&
          node.parent.children?.[0]?.openingElement?.name?.name === 'title'
```